Makefile.prow

```diff
@@ -1,92 +1,87 @@
-# warning: verify this inside a container with following:
-# docker run -it --rm -v `pwd`:`pwd` -w `pwd` golang:1.18 bash -c "make -f Makefile.prow ci"
-# 
-# otherwise unintended system changes may include, replacing your currently installed kubectl, binaries in GOBIN, files in GOPATH/src
-# 
-# If formatting needs updating you can run
+# Warning: Verify this inside a container with the following command:
+# podman run -it --rm -v `pwd`:`pwd`:Z -w `pwd` golang:1.22 bash -c "make -f Makefile.prow ci"
+#
+# Otherwise, unintended system changes may occur, including replacing your
+# currently installed kubectl, binaries in GOBIN, files in GOPATH/src
+#
+# To update formatting, run:
 # GOFLAGS=-mod=mod make update
 
-GOFLAGS=-mod=mod
-CONGEN_VERSION=0.14.0
-CODEGEN_VERSION=0.22.2
-PROWBIN=/tmp/prowbin
-# Name of this Makefile
-MAKEFILE=Makefile.prow
-# emulate as close as possible upstream ci target
+# Configuration Variables
+GOFLAGS := -mod=mod
+CONGEN_VERSION := 0.14.0
+CODEGEN_VERSION := 0.22.2
+GOLANGCI_VERSION := 1.57.2
+PROWBIN := /tmp/prowbin
+
+# GOPATH Setup
+GOPATH := $(shell go env GOPATH)
+GOBIN := $(GOPATH)/bin
+GOSRC := $(GOPATH)/src
+
+
 # upstream ci target: verify-modules verify all test
-#  we only need to modify verify, test, all to avoid docker usage
+# we need to modify verify, test, all to avoid usage of docker CLI
 .PHONY: ci
-ci:
-	@echo "go version is: $(shell go version)"
-	GOFLAGS=$(GOFLAGS) make verify-modules
-	make -f $(MAKEFILE) verify all test
+ci: verify-modules verify all test
+
+.PHONY: verify-modules
+verify-modules:
+	@echo "verify-modules target is a placeholder to use same targets as upstream CI"
```

Comment:
how this calls the target in the other Makefile?

Reply:
It does not, it's just a placeholder that does *nothing*, but the `make ci` target is exactly the same as upstream:

https://github.com/openshift/velero/blob/32e4c959cc1c4e355fc50e69acb6cffc32b74712/Makefile#L330

equals to our Makefile.prow target.

Reply:
Ok will update to call original upstream.

Reply:
will try to produce a test locally